init.lua

```diff
@@ -59,7 +59,7 @@ dofile(modpath.."/api.lua")
 dofile(modpath.."/internal.lua")
 dofile(modpath.."/callbacks.lua")
 dofile(modpath.."/register.lua")
-dofile(modpath.."/bags.lua")
+--dofile(modpath.."/bags.lua")
```

Comment:
Oh.
RREDesign added it in his commit
I added it because I was on creative branch. Should I remove it ?